### PR TITLE
WIP: Introduce CSV based dataframe to handle data lookup

### PR DESCRIPTION
**Pull request by Aniketh01**

### CODE REVIEW

File: data.csv [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Proto_list,Blacklist,Banwords
+#netsec,Pornhub,porn
+#DNS,xnxx,pornhub
+#DANE,,xvideo
+#DNSSEC,,sex
+#QUIC,,
```
Comment (ananthanandanan) [inline data.csv]: What does the double coma signify?

Comment (Aniketh01) [reply data.csv]: That's how a CSV file is written. As the full form says, **C**omma **S**eparated **V**alues :)

File: nawab_bot.py [hunk 2]
```diff
@@ -7,12 +7,15 @@
...
 import random
 from datetime import date
+import pandas as pd
 
 # Banned handles and words
 banned_accs =  []
 banned_words = []
 whitelist_accs = []
 
+data = pd.read_csv('data.csv')
```
Comment (iammarco11) [inline nawab_bot.py:14]: this data should be used below instead of opening the text file right?

File: data.csv [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#SNI,,
+#HTTP/0.9,,
+#HTTP0.9,,
+#HTTP2,,
+#HTTP/2,,
+#HTTP3,,
+#HTTP/3,,
+#TLS,,
+#TCP,,
+#UDP,,
```
Comment (iammarco11) [inline data.csv]: I think we can also the retweets ids in the same file

Comment (ananthanandanan) [reply data.csv]: Yeah

**Pull request merged**